zebra-network/src/peer_set/set.rs

```diff
@@ -504,7 +504,6 @@ where
     /// Checks if the minimum peer version has changed, and disconnects from outdated peers.
     fn disconnect_from_outdated_peers(&mut self) {
         if let Some(minimum_version) = self.minimum_peer_version.changed() {
-            // TODO: Remove when the code base migrates to Rust 2021 edition (#2709).
```

Comment:
Should the below line be removed also?

Reply:
Yes, but it needs a separate PR because it changes multiple lines, see PR #6737

Reply:
Yes, but it needs a separate PR because it changes multiple lines, see PR #6737

Reply:
Added mergify dependency on #6737